### PR TITLE
[UPDATE][K-P]'reworked keywords, fixed formatting, emphasis on vegan'

### PR DESCRIPTION
**Pull request by M-McCallum**
Updated keywords for recipes K - P with emphasis on "vegan". Also formatting. 

issue - https://github.com/alfg/opendrinks/issues/139

### CODE REVIEW

File: src/recipes/mango-lassi.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "Mango Lassi",
     "description": "This recipe comes from the cookbook author Madhur Jaffrey, who said, 'Mangoes are supposed to be good for the eyes.'",
-    "github": "parammittal16",
+    "github": "parammittal  16",
```
Comment (alfg) [inline src/recipes/mango-lassi.json]: Mistake here?

File: src/recipes/mango-lassi.json [hunk 2]
```diff
@@ -17,11 +17,6 @@
             "quantity": "",
             "measure": "",
             "ingredient": "One-Fourth cup fresh or canned mango pulp"
-        },
-        {
-            "quantity": "",
-            "measure": "",
-            "ingredient": "Three-fourth tablespoon sugar"
```
Comment (alfg) [inline src/recipes/mango-lassi.json]: Was this meant to be removed?

File: src/recipes/mango-lassi.json (hunk 1 above)
Comment (M-McCallum) [reply src/recipes/mango-lassi.json]: Must be, (I wasn't trying to change usernames) I'll fix and recommit.

File: src/recipes/mango-lassi.json (hunk 2 above)
Comment (M-McCallum) [reply src/recipes/mango-lassi.json]: Yes, I think was an accidental repeat of the ingredient. The author additionally indicated in the directions that you're to "sugar to taste" so the amount of sugar added before this step seems redundant anyway. I can re-commit with this added back in if necessary.

File: src/recipes/mango-lassi.json (hunk 1 above)
Comment (M-McCallum) [reply src/recipes/mango-lassi.json]: fix implemented with last commit.

File: src/recipes/mango-lassi.json (hunk 2 above)
Comment (alfg) [reply src/recipes/mango-lassi.json]: Thanks for the clarification. I would say leave the ingredient in so we have the data as part of the ingredient list. But if you can add the "Three-fourth" to the quantity, and "tablespoon" to the measure fields.

Comment (M-McCallum) [reply src/recipes/mango-lassi.json]: I'm actually working on another PR in which I go through A-P recipes and fix errors like empty measurement fields, strange white spaces, and misspellings. so for now I'll recommit with the aforementioned fixes and then submit an additional PR later on for more fixes.

Comment (alfg) [reply src/recipes/mango-lassi.json]: OK, sounds good.

**Pull request merged**